### PR TITLE
removing parameter hardcoded. add the timeout of G5k in the config

### PR DESCRIPTION
**Pull request by martinezmatias**

### CODE REVIEW

File: script/config.py [hunk 1]
```diff
@@ -17,3 +17,5 @@
 GRID5K_MAX_NODE = 50
 
 TOOL_TIMEOUT = "120"
+#Format HH:MM
+GRID5K_TIME_OUT = "2:15"
```
Comment (tdurieux) [inline script/config.py]: This property should be automatically computed from `TOOL_TIMEOUT`

Comment (martinezmatias) [reply script/config.py]: Yes, good point. It should take TOOL_TIMEOUT and add it an X%

**Comment by martinezmatias** [comment]
Hi @tdurieux 
I updated the code. WDYT?

**Comment by tdurieux** [comment]
It seems ok. Maybe can you add a comment that says that the overhead is a percentage of the timeout?

**Comment by martinezmatias** [comment]
Updated

**Pull request merged**